.eslintrc.yaml

```diff
@@ -135,16 +135,17 @@ rules:
   }]
   no-tabs: error
   no-trailing-spaces: error
+  object-curly-spacing: [error, always]
```

Comment:
You can reduce the churn slightly and conform more closely to the house style by doing this instead:

```yaml
  object-curly-spacing: [error, always, {objectsInObjects: false}]
```

That flags 2144 problems in the current code base, instead of the 2183 flagged with the PR as it is. That's not a huge reduction, but every little bit helps to make backporting less awful.